src/demos/DlSliderDemo.vue

```diff
@@ -58,6 +58,14 @@ export default defineComponent({
         handleChange(value: number) {
             console.log(`@@@ handling change ${value}`)
             this.events[1] = `@@@ handling change ${value}`
+        },
+        handleFocus(event: Event) {
+            console.log(`@@@ handling focus ${event}`)
+            this.events[1] = `@@@ Input Focus `
```

Comment:
What is that for ?

Reply:
Could you elaborate , I didn't understand the question.

Reply:
I meant to ask what's the purpose of adding this string to the events array, because it looks like a debugging string.
But now I see it's implemented that way in previous cases, so never mind.